src/mmw/apps/modeling/views.py

```diff
@@ -202,13 +207,29 @@ def _initiate_tr55_job_chain(model_input, job_id):
 
 @decorators.api_view(['GET'])
 @decorators.permission_classes((AllowAny, ))
-def district(request, id=None, state=None):
-    if id:  # query by unique id
-        district = get_object_or_404(District, id=id)
-        geojson = json.loads(district.polygon.geojson)
+def boundary_layers(request, table_id=None, obj_id=None):
+    if not table_id or not obj_id:
```

Comment:
Seems like we require `table_id` for either conditional, should this be an `and` and then another conditional to return a bad request response if it's not provided?